### PR TITLE
Add comment about group merge in yaml inventory example

### PR DESCRIPTION
**Pull request by emonty**
##### ISSUE TYPE
Docs Pull Request

##### COMPONENT NAME
N/A

##### SUMMARY
From reading the example it wasn't fully clear what was happening with
the hosts in the webserver child group in the testing group definition.
There is a very clear answer though - so just spell it out in case
anyone else is confused.

### CODE REVIEW

**Comment by acozine** [comment]
Thanks @emonty for reviewing the documentation. I find these changes a bit confusing, though. Could you describe the experience that led you to suggest this change? Based on the current documentation, what behavior did you expect? And what behavior did you get instead?

File: examples/hosts.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #   - Hosts must be specified in a group's hosts:
 #     and they must be a key (: terminated)
 #   - groups can have children, hosts and vars keys
+#   - a group hostlist is definable anywhere a group is. multilple hostlists for a group should be merged
```
Comment (bcoca) [inline examples/hosts.yaml]: I would rephrase:

- groups are unique and global, so defining a group in multiple locations aggregates all the data to the global name . So if you define a group as a child of 2 different groups, it will be the child of both, any hosts and variables assigned will not be dependent on the parents, they will all be associated with the group.

as its not just the hostlist, its vars and child group relationships

**Comment by acozine** [comment]
@emonty @bcoca I updated this PR and incorporated the suggested changes - does it look okay to you?

**Review by bcoca — APPROVED** [review]
